src/Simulator/IncrementalRedstoneSimulator/RedstoneWireHandler.h

```diff
@@ -35,7 +35,7 @@ class cRedstoneWireHandler : public cRedstoneHandler
 	const cVector3iArray GetTerracingConnectionOffsets(const Vector3i & a_Position)
 	{
 		cVector3iArray RelativePositions;
-		bool IsYPTerracingBlocked = cBlockInfo::IsSolid(m_World.GetBlock(a_Position + OffsetYP()));
+		bool IsYPTerracingBlocked = cBlockInfo::IsSolid(m_World.GetBlock(a_Position + OffsetYP())) ? !cBlockInfo::IsTransparent(m_World.GetBlock(a_Position + OffsetYP())) : false;
```

Comment:
Wouldn't a simple `&&` between the two properties work as intended?
`IsSolid() && !IsTransparent()`